pkg/network/multus.go

```diff
@@ -69,8 +70,14 @@ func renderMultusConfig(manifestDir, defaultNetworkType string, useDHCP bool, us
 	data.Data["WhereaboutsImage"] = os.Getenv("WHEREABOUTS_CNI_IMAGE")
 	data.Data["EgressRouterImage"] = os.Getenv("EGRESS_ROUTER_CNI_IMAGE")
 	data.Data["RouteOverrideImage"] = os.Getenv("ROUTE_OVERRRIDE_CNI_IMAGE")
-	data.Data["KUBERNETES_SERVICE_HOST"] = apihost
-	data.Data["KUBERNETES_SERVICE_PORT"] = apiport
+	hsc := hypershift.NewHyperShiftConfig()
+	if hsc.Enabled && bootstrapResult.Infra.PlatformType == configv1.IBMCloudPlatformType {
```

Comment:
I assumed that this should be applicable to all hypershift deployments, not only IBMCloudPlatformType?

Reply:
Ah, I now understand what you meant here: https://github.com/openshift/cluster-network-operator/pull/2288/files#diff-2c4ca959f28adcdad43be768016a18842630345bec246de204c7c9d8a541c0f5R213-R214
Thanks!
The only thing that seems to be missing in the hypershift PR is only missing a valid jira label.

Reply:
Considering that the hypershift PR is merged can we remove the `IBMCloudPlatformType` in this PR?

Reply:
I'm worried about your existing production fleet: I would like to get confirmation that all operators are updated before we do so......